lib/cubits/user/user_cubit.dart

```diff
@@ -1,22 +1,38 @@
 import 'package:bloc/bloc.dart';
 import 'package:coffeecard/data/repositories/v1/account_repository.dart';
+import 'package:coffeecard/data/repositories/v1/programme_repository.dart';
 import 'package:coffeecard/models/account/user.dart';
 import 'package:equatable/equatable.dart';
 
 part 'user_state.dart';
 
 class UserCubit extends Cubit<UserState> {
   final AccountRepository accountRepository;
+  final ProgrammeRepository programmeRepository;
 
-  UserCubit(this.accountRepository) : super(UserLoading());
+  UserCubit(this.accountRepository, this.programmeRepository)
+      : super(UserLoading());
 
   Future<void> fetchUserDetails() async {
     emit(UserLoading());
 
     final either = await accountRepository.getUser();
 
     if (either.isRight) {
-      emit(UserLoaded(either.right));
+      var user = either.right;
+
+      final programmes = await programmeRepository.getProgramme();
```

Comment:
Any way that we could save this data somewhere, in order to reuse it for when a user wants to update their current programme?

Reply:
yes that we should be able to handle. Opened #187